tools/testing/test_selections.py

```diff
@@ -290,6 +290,17 @@ def export_S3_test_times(test_times_filename: Optional[str] = None) -> Dict[str,
     return test_times
 
 
+# TODO refactor this and unify with tools.stats.export_*
+def export_target_det_list(selected_test: List[Any], target_det_filename: str) -> None:
+    if target_det_filename is not None:
+        print(f'Exporting target determination result to {target_det_filename}.')
+        if os.path.exists(target_det_filename):
+            print(f'Overwriting existent file: {target_det_filename}')
+        with open(target_det_filename, 'w+') as file:
+            json.dump(selected_test, file, indent='    ', separators=(',', ': '))
+            file.write('\n')
```

Comment:
do we export it to artifact or other place, wondering how can we read the file content once saved

Reply:
this is only meant for local use. At this point, this PR only provides this capability to export to a file (similar to export_slow_test), it does not enable using it on CI, which will be done in a separate PR.